keps/prod-readiness/sig-node/2570.yaml

```diff
@@ -1,3 +1,3 @@
 kep-number: 2570
-alpha:
+beta:
```

Comment:
leave alpha in and add beta as a separate approval field - that way we have history of who approved each stage